CMakeLists.txt

```diff
@@ -455,15 +469,8 @@ SET(BUILD_SHARED_LIBS ON)
 
 ## HEADER/LIBRARY/OTHER CHECKS ##
 # First, required stuff
-if(MSVC)
-    find_package(SndFile CONFIG REQUIRED)
-else()
-    find_library(LIBSNDFILE_LIBRARY NAMES sndfile libsndfile-1 libsndfile)
-
-    if(NOT LIBSNDFILE_LIBRARY)
-        message(FATAL_ERROR "Csound requires the sndfile library")
-    endif()
-endif()
+# provided by vcpkg
+find_package(SndFile CONFIG REQUIRED)
```

Comment:
The else part is for non VCPKG builds. You can see the libraries have multiple potential names. This will break the current builds for most mac/linux users.